plenum/server/node.py

```diff
@@ -3505,7 +3505,12 @@ def reportSuspiciousNode(self,
                                      Suspicions.PPR_PLUGIN_EXCEPTION,
                                      Suspicions.PPR_SUB_SEQ_NO_WRONG,
                                      Suspicions.PPR_NOT_FINAL,
-                                     Suspicions.PPR_WITH_ORDERED_REQUEST)):
+                                     Suspicions.PPR_WITH_ORDERED_REQUEST,
+                                     Suspicions.PPR_AUDIT_TXN_ROOT_HASH_WRONG,
+                                     Suspicions.PPR_BLS_MULTISIG_WRONG,
+                                     Suspicions.PPR_TIME_WRONG,
+                                     Suspicions.DUPLICATE_PPR_SENT
```

Comment:
After more thinking, I think we do not need to do view change if we have `DUPLICATE_PPR_SENT`, since we may have a gap in PP, request it, and then receive the original. So, we have  2 same PPs, but this is fine, and doesn't mean that the Primary is malicious.

Reply:
ok